### ISSUE TITLE
Fix State Field Case Sensitivity for Ambassador Sign-Up

### ISSUE THREAD

**Issue by ajones620** [colab-coop/hello-voter#121]
# Summary
As an Ambassador going through the sign-up workflow, I do not want to be blocked by entering my state in lowercase format so that I do not experience any friction in the sign-up process.

# Requirements
* When I am signing up to be an Ambassador, if I enter either or both characters for my state in lowercase format, I should not see an error
 * Currently the error message for using any lowercase characters is "Sorry, but state employment laws don't allow us to pay Voting Ambassadors in your state."

Current page/error message for reference:
<img width="719" alt="Screen Shot 2020-09-25 at 6 49 45 PM" src="https://user-images.githubusercontent.com/8608070/94345311-d84afa80-ffd9-11ea-9bb4-2e0ca3eede0a.png">

**Comment by joshmaker** [comment]
I wonder if this change to HelloVoter would fix this issue:

```diff
diff --git a/server/app/services/ambassadors.js b/server/app/services/ambassadors.js
index 36b63662..f77aec95 100644
--- a/server/app/services/ambassadors.js
+++ b/server/app/services/ambassadors.js
@@ -59,8 +59,8 @@ async function signup(json) {
     throw new ValidationError("Our system doesn’t recognize that phone number. Please try again.");
   }

-  let allowed_states = ov_config.allowed_states.split(',');
-  if (allowed_states.indexOf(json.address.state) === -1) {
+  let allowed_states = ov_config.allowed_states.toUpperCase().split(',');
+  if (allowed_states.indexOf(json.address.state.toUpperCase()) === -1) {
     throw new ValidationError("Sorry, but state employment laws don't allow us to pay Voting Ambassadors in your state.")
   }
```

**Comment by joshmaker** [comment]
I think this ticket should be resolved now that colab-coop/HelloVoter#48 has been merged. Basically, I convert everything to uppercase on the backend that way the API can receive lower, upper, or mixed case state abbreviations and it won't matter.

**Comment by shauminn-colab** [comment]
That's so awesome @joshmaker - thank you! ill test it out on stage2 👍
